### PR TITLE
Display loqusdb obs only in current case

### LINKED ISSUE #3469: Note if individual with loqus match is in the current case
It can be confusing with loqusdb counts of 1, especially as occasionally 0 counts can appear for the same individual due to e.g. low quality. A flag on the variant page if the count is for the same case or other.

Note on prio: it is most often not critical to know if the individual variant is found in one more in 7000, but perhaps with a really unusual condition, that is borderline between two different institutes patient groups it would be unclear?

### PR DESCRIPTION
**Pull request by northwestwitch**
- Fix #3469 
- Removes some old unused code that causes this:
 
image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. Install current version branch on scout stage (cg-vm1) and not that this variant has observations but no cases shown: https://scout-stage.scilifelab.se/cust000/NA24143-200ng/fcb3ee295a4baae271f9cb97f6834930
2. Switch to this branch and check that it's shown that observation is only in this case

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by mikaell
- [x] tests executed by mikaell

### CODE REVIEW

**Comment by northwestwitch** [comment]
Before:

image


After:

image

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 1]
```diff
@@ -180,31 +180,15 @@
...
+                    observed only in this case
                   {% endif %}
               </td>
             </tr>
           {% endfor %}
         </tbody>
       </table>
     </div>
-
-    <div class="card-footer">
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:191]: This was not used

**Review by mikaell — COMMENTED** [review]
Code looks good. I've capitalised "Observed only in this case" for readability.

Just need to test it.

**Comment by northwestwitch** [comment]
@mikaell could it be that you forgot to approve this one?

**Comment by mikaell** [comment]
> @mikaell could it be that you forgot to approve this one?

Got caught in the rain. Now tested and approved.

**Pull request merged**